### PR TITLE
Remove install files for CentOS7, RHEL 7, Debian 10, SLES 12 for Connectors

### PR DESCRIPTION
**Pull request by dwicinas**
Update templates and modify generated files for connector products

### CODE REVIEW

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx [hunk 1]
```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](jdbc_sles_15)
-
-- [SLES 12](jdbc_sles_12)
```
Comment (nidhibhammar) [inline product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx:31]: Need to remove SLES 12 from navigation list as well.

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_x86_64/index.mdx [hunk 2]
```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
...
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](jdbc_ubuntu_22)
 
 - [Ubuntu 20.04](jdbc_ubuntu_20)
 
 - [Debian 11](jdbc_debian_11)
-
-- [Debian 10](jdbc_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_x86_64/index.mdx:67]: Remove all the deprecated platform topics from navigation as well

File: product_docs/docs/ocl_connector/14/installing/linux_ppc64le/index.mdx [hunk 3]
```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](ocl_sles_15)
-
-- [SLES 12](ocl_sles_12)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/14/installing/linux_ppc64le/index.mdx:31]: Remove from navigation

File: product_docs/docs/ocl_connector/14/installing/linux_x86_64/index.mdx [hunk 4]
```diff
@@ -48,18 +44,12 @@ Operating system-specific install instructions are described in the correspondin
...
 
-- [SLES 12](ocl_sles_12)
-
 ### Debian and derivatives
 
 - [Ubuntu 20.04](ocl_ubuntu_20)
 
 - [Debian 11](ocl_debian_11)
-
-- [Debian 10](ocl_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/14/installing/linux_x86_64/index.mdx:65]: Remove all from navigation as well.

File: product_docs/docs/ocl_connector/16/installing/linux_ppc64le/index.mdx [hunk 5]
```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](ocl_sles_15)
-
-- [SLES 12](ocl_sles_12)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/16/installing/linux_ppc64le/index.mdx:31]: Remove from navigation

File: product_docs/docs/ocl_connector/16/installing/linux_x86_64/index.mdx [hunk 6]
```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
...
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](ocl_ubuntu_22)
 
 - [Ubuntu 20.04](ocl_ubuntu_20)
 
 - [Debian 11](ocl_debian_11)
-
-- [Debian 10](ocl_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/16/installing/linux_x86_64/index.mdx:67]: Remove all from navigation as well

File: product_docs/docs/ocl_connector/15/installing/linux_x86_64/index.mdx [hunk 7]
```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
...
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](ocl_ubuntu_22)
 
 - [Ubuntu 20.04](ocl_ubuntu_20)
 
 - [Debian 11](ocl_debian_11)
-
-- [Debian 10](ocl_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/15/installing/linux_x86_64/index.mdx:67]: Remove all from navigation

File: product_docs/docs/ocl_connector/15/installing/linux_ppc64le/index.mdx [hunk 8]
```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](ocl_sles_15)
-
-- [SLES 12](ocl_sles_12)
```
Comment (nidhibhammar) [inline product_docs/docs/ocl_connector/15/installing/linux_ppc64le/index.mdx:31]: Remove from navigation

File: product_docs/docs/odbc_connector/13/installing/linux_ppc64le/index.mdx [hunk 9]
```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](odbc_sles_15)
-
-- [SLES 12](odbc_sles_12)
```
Comment (nidhibhammar) [inline product_docs/docs/odbc_connector/13/installing/linux_ppc64le/index.mdx:31]: Same comment as above

File: product_docs/docs/odbc_connector/13/installing/linux_x86_64/index.mdx [hunk 10]
```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
...
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](odbc_ubuntu_22)
 
 - [Ubuntu 20.04](odbc_ubuntu_20)
 
 - [Debian 11](odbc_debian_11)
-
-- [Debian 10](odbc_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/odbc_connector/13/installing/linux_x86_64/index.mdx:67]: Same comment as above

File: product_docs/docs/odbc_connector/16/installing/linux_x86_64/index.mdx [hunk 11]
```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
...
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](odbc_ubuntu_22)
 
 - [Ubuntu 20.04](odbc_ubuntu_20)
 
 - [Debian 11](odbc_debian_11)
-
-- [Debian 10](odbc_debian_10)
```
Comment (nidhibhammar) [inline product_docs/docs/odbc_connector/16/installing/linux_x86_64/index.mdx:67]: Same comment as above

**Review by nidhibhammar — COMMENTED** [review]
Left few comments

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx (hunk 1 above)
Comment (dwicinas) [reply product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx:31]: done

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_x86_64/index.mdx (hunk 2 above)
Comment (dwicinas) [reply product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_x86_64/index.mdx:67]: done

File: product_docs/docs/ocl_connector/14/installing/linux_ppc64le/index.mdx (hunk 3 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/14/installing/linux_ppc64le/index.mdx:31]: done

File: product_docs/docs/ocl_connector/14/installing/linux_x86_64/index.mdx (hunk 4 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/14/installing/linux_x86_64/index.mdx:65]: done

File: product_docs/docs/ocl_connector/16/installing/linux_ppc64le/index.mdx (hunk 5 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/16/installing/linux_ppc64le/index.mdx:31]: done

File: product_docs/docs/ocl_connector/16/installing/linux_x86_64/index.mdx (hunk 6 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/16/installing/linux_x86_64/index.mdx:67]: done

File: product_docs/docs/ocl_connector/15/installing/linux_x86_64/index.mdx (hunk 7 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/15/installing/linux_x86_64/index.mdx:67]: done

File: product_docs/docs/ocl_connector/15/installing/linux_ppc64le/index.mdx (hunk 8 above)
Comment (dwicinas) [reply product_docs/docs/ocl_connector/15/installing/linux_ppc64le/index.mdx:31]: done

File: product_docs/docs/odbc_connector/13/installing/linux_ppc64le/index.mdx (hunk 9 above)
Comment (dwicinas) [reply product_docs/docs/odbc_connector/13/installing/linux_ppc64le/index.mdx:31]: done

File: product_docs/docs/odbc_connector/13/installing/linux_x86_64/index.mdx (hunk 10 above)
Comment (dwicinas) [reply product_docs/docs/odbc_connector/13/installing/linux_x86_64/index.mdx:67]: done

File: product_docs/docs/odbc_connector/16/installing/linux_x86_64/index.mdx (hunk 11 above)
Comment (dwicinas) [reply product_docs/docs/odbc_connector/16/installing/linux_x86_64/index.mdx:67]: done

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx [hunk 12]
```diff
@@ -11,7 +11,6 @@ redirects:
 
 navigation:
   - jdbc_rhel_9
-  - jdbc_rhel_8
```
Comment (nidhibhammar) [inline product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx:14]: Why are we removing rhel_8?

**Review by moazzumali — APPROVED** [review]

File: product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx (hunk 12 above)
Comment (dwicinas) [reply product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx]: I'm not sure why that happened. I thought it was the templates do that, but that doesn't make any sense. I added it back in.

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**